### PR TITLE
Stop using log app in CI links and improve dashboard link

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

The log app is taking too much time to load, so revert to
the dashboard even after a CI job is completed, at least
until we have a working alternative.

Improve the link to the dashboard to point to a pipelinerun
(when available) and to the specific pipeline task instead
of always pointing to the the taskrun directly.

This gives visibility into the checks that were executed too.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

/kind cleanup

### CODE REVIEW

**Review by vdemeester — COMMENTED** [review]
/lgtm

File: tekton/resources/ci/github-template.yaml [hunk 1]
```diff
@@ -48,25 +56,26 @@ spec:
...
-              else:
-                check_status = "error"
+            if ("$(tt.params.checkResult)" == "True"):
+              check_status = "success"
+            elif ("$(tt.params.checkResult)" == "False"):
+              check_status = "failure"
             else:
-              logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/tektonci/taskruns/$(tt.params.taskRunName)'
+              check_status = "error"
+            logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/#/namespaces/$(params.taskRunNamespace)/' + resource_path
```
Comment (AlanGreene) [inline tekton/resources/ci/github-template.yaml:78]: Looks like an extra `/#/namespaces` in the URL that should be removed:

```suggestion
            logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/$(params.taskRunNamespace)/' + resource_path
```

Comment (afrittoli) [reply tekton/resources/ci/github-template.yaml:78]: Oh, nice catch, thank you!

**Review by vdemeester — APPROVED** [review]
/meow

**Pull request merged**